realm/realm-library/src/main/java/io/realm/CollectionUtils.java

```diff
@@ -148,6 +171,31 @@ static Mixed copyToRealmIfNeeded(BaseRealm realm, Mixed mixed) {
         }
     }
 
+    /**
+     * Called by both list, sets and dictionary operators to copy a RealmModel to Realm in case it has
+     * been deemed necessary.
+     *
+     * @param baseRealm The Realm instance to copy the object to.
+     * @param objects   collection of object to copy.
+     * @param <E>       The RealmModel type.
+     * @return the copied object
+     */
+    public static <E extends RealmModel> Collection<E> copyToRealm(BaseRealm baseRealm, Collection<E> objects) {
```

Comment:
Maybe be explicit about it returning a list, as people could think that it was in fact the same type of collection as the argument.

Reply:
Actually it is not needed forgot to remove it